async function throwError() {
    throw Error('hello');
}
